colossalai/utils/checkpoint/module_checkpoint.py

```diff
@@ -71,22 +71,23 @@ def save_checkpoint(path: str,
         dist.barrier()
 
 
-def load_checkpoint(path,
+def load_checkpoint(path: str,
                     epoch: int,
                     model: torch.nn.Module,
                     optimizer: Optional[ColossalaiOptimizer] = None,
                     lr_scheduler: torch.optim.lr_scheduler._LRScheduler = None,
+                    load_state_dict_kwargs: Optional[Dict] = None,
                     *args,
                     **kwargs):
```

Comment:
It can be quite confusing to the user if there are two kwargs

Reply:
Yeah, there are two functions requiring kwargs.
I think kwargs is conveninent for users.
Should we remove kwargs and add two dict for paramters passing in?

Reply:
I think it will be better.

Reply:
Done.